packages/js/app/src/components/scaleComponent.tsx

```diff
@@ -0,0 +1,15 @@
+import { Dimensions } from 'react-native';
```

Comment:
What is scaleComponent and why is it needed?

Reply:
The input boxes were too big for aspect ratios like the iphone 8
scale component scales width or height with respect to the width and height of the screen
Should I get rid of it?

Reply:
CSS *should* be expressive enough to do this, try to do it with just CSS

Reply:
I've tried. I will try again

Reply:
Sorry, I was being dumb. I am fixing it